pinot-common/src/main/java/org/apache/pinot/common/function/scalar/StringFunctions.java

```diff
@@ -443,4 +446,28 @@ public static boolean contains(String input, String substring) {
   public static int strcmp(String input1, String input2) {
     return input1.compareTo(input2);
   }
+
+  /**
+   *
+   * @param input plaintext string
+   * @return url encoded string
+   * @throws UnsupportedEncodingException
+   */
+  @ScalarFunction
+  public static String encodeUrl(String input)
+      throws UnsupportedEncodingException {
+      return URLEncoder.encode(input, StandardCharsets.UTF_8.toString());
```

Comment:
```suggestion
      return URLEncoder.encode(input, StandardCharsets.UTF_8);
```

Reply:
The java8 compilation will fail if we don't add `.toString()` 
https://github.com/apache/pinot/runs/5632328340?check_suite_focus=true